src/bots/monitor/index.js

```diff
@@ -0,0 +1,44 @@
+import { attachmentIsAllowed } from './validator';
+import client from '../../client';
+import { defaultStrings } from './constants';
+import { get } from '../../environment';
+
+client.on('messageCreate', async (message) => {
+  const { author } = message;
+
+  // Run routine based on id
+  switch (message.channelId) {
+    // Career questions channel
+    case get('CAREER_QUESTIONS_CHANNEL_ID'):
+      // Create a new thread under each message sent
+      message.startThread({
+        name: defaultStrings.careerQuestion(author.username)
+      });
+      break;
+
+    // Resume review channel
+    case get('RESUME_REVIEW_CHANNEL_ID'):
+      // Verify a resume is attached to message
+      if (message.attachments.size === 0) {
+        author.send(defaultStrings.noResume);
+        message.delete();
+        return;
+      }
+
+      // Verify that all attachments are in valid format for a resume
+      if (!message.attachments.every((file) => attachmentIsAllowed(file))) {
+        author.send(defaultStrings.invalidFormat);
+        message.delete();
+        return;
+      }
```

Comment:
The DM should include the text the user posted and which got deleted. This way, if they had written a massive wall-of-text, it's not lost.

Reply:
Great idea, I'll add that to the next revision